addons/common/XEH_postInit.sqf

```diff
@@ -254,12 +254,25 @@ GVAR(commonPostInited) = true;
 [{
     // If post inits are not ready then wait
     if !(SLX_XEH_MACHINE select 8) exitWith {};
+
     // If settings are not initialized then wait
-    if !(GVAR(SettingsInitialized)) exitWith {};
+    if (isNil QGVAR(settings)) exitWith {
+        diag_log text format["[ACE] Waiting on settings from server"];
```

Comment:
Well, I noticed that, but I decided leaving that spam on the first version should be good, to be able to easily check how long the settings are delaying. I propose to remove it on the next update.